README.md

```diff
@@ -0,0 +1,77 @@
+This is a dashboard for tracking outstanding bugs in **Ironic**.
+
+**Ironic** is one of the software components in OpenStack that fully manages baremetal infrastructure; it discovers bare-metal modes; catalogs them in a management database and manages the entire server lifecycle; from enrolling to provisioning, maintenance and decomissioning.
```

Comment:
nit: bare-metal vs baremetal: Perhaps make it baremetal, keeping it consistent everywhere in the repository would be nice.
Also, do you mean "...it discovers baremetal nodes..." instead of modes?

Reply:
Good eye, I indeed meant "nodes". Consistency fix implemented.